Shared/AppConstants.swift

```diff
@@ -176,9 +176,9 @@ public struct AppConstants {
         #if MOZ_CHANNEL_RELEASE
             return false
         #elseif MOZ_CHANNEL_BETA
-            return false
+            return true
```

Comment:
@st3fan I am not familiar with what step Leanplum gets turned on, does it make sense to enable for these channels?